samples/gitchurn/src/gitChurnMain/kotlin/GitChurn.kt

```diff
@@ -35,11 +35,14 @@ fun main(args: Array<String>) {
 private fun calculateChurn(workDir: String, limit: Int) {
     println("Opening…")
     val repository = git.repository(workDir)
-    val map = mutableMapOf<String, Int>()
+    //val map = mutableMapOf<String, Int>()
+    val namedMap = mutableMapOf<Pair<String, String>, Int>()
```

Comment:
The code becomes unreadable at this point.
To figure out `namedMap` meaning one have to refer to the code populating this map below.
When it is possible, `namedMap` should describe its purpose clearly by itself.

To achieve that, it makes sense to 
1.  Declare a `data class` instead of using `Pair`
2. Rename `namedMap` to something more meaningful.